### ISSUE TITLE
An 'on rotation' question ~ not an issue

### ISSUE THREAD

**Issue by wm-j-ray** [scalessec/CSLinearLayoutView#3]
I'm a using CSLinearLayoutView in vertical mode with device in portrait orientation.  When I rotate the device to landscape I would like the view(s)/items to expand horizontally, i.e. keeping the same left and right margins they had in portrait.

I've fooled around with the linearLayoutView.autoresizingMask but had no luck.  (I'm not even sure I'm looking in the right place)

Can you give me a pointer as to where to start?

Thank you so much for your upload.  Very cleverly done.

**Comment by scalessec** [comment]
You're right, this is a bug. The autoresizingMask on the linear layout view itself works as expected, but the layout items' views should respect their own individual autoresizing masks when the fillMode is set to `CSLinearLayoutItemFillModeNormal`. I'll investigate...

**Comment by wm-j-ray** [comment]
Anything? Tell me where to start and I'll take a whack at it.
Please let me know if you have any any thoughts as to where I could start to fix this.

Thanx in advance.
